lesson_06/task_3/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"text/template"
+	"time"
+)
+
+type response struct {
+	Cookie []string
+}
+
+func handlerCookie(w http.ResponseWriter, r *http.Request) {
+	// Web server, responds to “/”
+	if r.URL.Path != "/" {
+		http.Error(w, "404 not found.", http.StatusNotFound)
+		return
+	}
+
+	switch r.Method {
+	case "GET":
```

Comment:
It is better to use here predefined constants from "http" package
see it here: https://pkg.go.dev/net/http#pkg-constants